src/app/containers/MediaPageBlocks/Blocks/Unknown/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+/* eslint-disable react/no-danger */
+const UnknownBlockContainer = ({ blockType }) => (
+  <div
+    dangerouslySetInnerHTML={{ __html: `<!-- unknown block ${blockType} -->` }}
```

Comment:
benefit of adding an the type to this comment is probably not worth the xss risk

Reply:
Hmm, I get your point, but the data can only be from the page data, is that much of a risk?

Reply:
doesn't the page data come from external systems though?  cps / ares

the risk is certainly small, but the benefit is even smaller :)

Reply:
I agree with Ryan on this